cfgov/v1/static/js/admin/ask_cfpb_tips.js

```diff
@@ -26,22 +28,94 @@
           buttonset.append(buttonElement);
           
           buttonElement.on('click', function (event) {
+
+
             var sel = window.getSelection();
-            var range = sel.getRangeAt(0).cloneRange();
-            var contents = range.cloneContents();
-            var elem = document.createElement("aside");
-            elem.className = "answer-module";
-            elem.appendChild(contents);
-            range.deleteContents();
-            range.insertNode(elem);
-
-            $(range.commonAncestorContainer).find('*').each(function() {
-              if ($(this).is(':empty') || $(this).text() === '') {
-                $(this).remove();
+            var node = sel.baseNode;
+            
+            var tip = $(node).closest( '.' + baseClass )[0];
```

Comment:
I could write this as plain javascript, but since it's part of a jquery plugin it seems acceptable to use jquery convenience methods.